### PR TITLE
Complete Todo List Solution

### PR DESCRIPTION
**Pull request by whilestevego**
My PR!

😈 🤠 😎

### CODE REVIEW

**Comment by yogiduzit** [comment]
Pull Request Approved

**Comment by whilestevego** [comment]
My comments and things

**Comment by tspoon6** [comment]
Wow such code!

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # CodeCore Homework Repo
 
 This is a repo of all my homework submissions for Codecore.
+
+## Submissions
```
Comment (enitefall88) [inline readme.md]: /////ddc

File: readme.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
 # CodeCore Homework Repo
 
 This is a repo of all my homework submissions for Codecore.
+
```
Comment (JJG-Development) [inline readme.md]: roflcopter

File: readme.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
...
+## Submissions
+
+1. Todo List
```
Comment (whilestevego) [inline readme.md]: Nice use of a ordered list 👍

**Comment by yogiduzit** [comment]
The more I C, the less I C

File: todo-list/solution.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const list = []
+
+function addToList(thing, list) {
+  list.push(thing)
+}
```
Comment (whilestevego) [inline todo-list/solution.js]: Needs more push

Comment (yogiduzit) [reply todo-list/solution.js]: push --hard

**Pull request merged**